modules/document_repository/test/TestPlan.md

```diff
@@ -8,34 +8,37 @@
+6. Edit document properties and save. Check that changes were saved properly.
    [Manual Testing]
-7. Download a file and check content.
+7. Check that the breadcrumbs on the main page and the Edit page work.
    [Manual Testing]
-8. Add files in repository to test search according to file name, file type, version, site and uploaded by.
+8. Download a file and check the content.
    [Manual Testing]
-9. Check that expand/collapse works for non-empty categories. Check that when performing a search, the categories
-   containing the resulting files (and only those) are shown in a list format
+9. Add files in the repository to test search according to file name, file type, version, site and uploaded by.
    [Manual Testing]
-10. Check that if a category contains special characters (e.g. space, dot or comma) it can be expanded.
+10. Check that when performing a search without global filtering, the categories
+   containing the resulting files (and only those) are shown in a list format. 
    [Manual Testing]
-11. Add two files with the same name. You should get a 'duplicate file name' error.
+11. Check that when performing a search with global filtering, all files matching the filtered case 
+   (regardless of category) are shown in a list format. 
    [Manual Testing]
-12. Edit a file in the repository: check that “Date Uploaded” date is updated.
-    [Manual Testing]
-    [Manual Testing]
+12. Check that if a category contains special characters (e.g. space, dot or comma) it can be expanded.
+   [Manual Testing]
+13. Add two files with the same name. You should get a 'duplicate file name' error.
+   [Manual Testing]
+14. Edit a file in the repository: check that “Date Uploaded” date is updated.
     [Manual Testing]
-13. Check that preference “Receive Document Repository Notifications” works.
+15. Check that the preference “Receive Document Repository Notifications” works.
     [Manual Testing]
-14. Check that performance is OK with large number of files.
+16. Check that the performance is OK with large number of files.
     [Manual Testing]
-15. Check that if you have the 'Receive document repository notification' activated you will receive an email each 
+17. Check that if you have the 'Receive document repository notification' activated you will receive an email each 
     time one of the following event occurs:
        - Addition, deletion or modification of a file (by a user other than yourself)
        - Addition of a category (by a user other than yourself)
     Also check that the www address contained in the notification email is correct.
     [Manual Testing]
-16. To upload large files, check that the following settings are in php.ini (default: /etc/php/{php_version}/apache2/php.ini):
+18. To upload large files, check that the following settings are in php.ini (default: /etc/php/{php_version}/apache2/php.ini):
       session.gc_maxlifetime 10800
       max_execution_time  10800
       upload_max_filesize 1020M
```

Comment:
Hi @jesscall I'm guessing `post_max_size` in the php.ini should be configured as well.
`upload_max_filesize` is the limit of any single file.
`post_max_size` is the limit of the entire body of the request, which could include multiple files.

Reply:
@maltheism  Thanks for the suggestion! I checked what was written in the [document_repository README](https://github.com/aces/Loris/blob/4155ea37030eecc913701b1aebd81e0798916dbb/modules/document_repository/README.md) and re-iterated those values.